eprihlaska/views.py

```diff
@@ -1007,7 +1007,7 @@ def admin_ais_test():
     ctx = create_context(cosign_cookies,
                          origin='ais2.uniba.sk')
     # Do log in
-    ctx.request_html('/ais/login.do', method='POST')
+    ctx.request_html('/ais/loginCosign.do', method='POST')
```

Comment:
I would say this could go to settings.

Reply:
That's indeed true but at the end of the day, these values are basically hardcoded in AIS, so I think we can live with these values being hardcoded as well :)

Reply:
Not sure what "hardcoded in AIS" means, but we are using it on 2 places here.

Reply:
What I meant was that `loginCosign.do` is extremely unlikely to change and so extracting it to config does not make too much sense to me (just like we did not extract `login.do` to its own config variable).

But if you think it's worth it, I am happy to refactor this and put it to config :slightly_smiling_face: